script/build

```diff
@@ -8,6 +8,9 @@
 # With `test`, runs tests instead.
 
 set -e
+source script/check_go_version
+
+check_go_version
```

Comment:
We can. But ...

`script/bootstrap` doesn't run if the user already has bin/cucumber in their working directory.

The scenario I'm trying to avoid is this:
1. Developer builds with go1.6 using `make test-all`
2. Developer does a `git pull`
3. Developer does another `make test-all`

At this point, the tests all pass, but the developer has a version of hub that is susceptible to following redirects and forwarding auth headers. If they use it, they're open to a security risk, and they had no way to know that they shouldn't have been building with an older version of go.